MdsSupportingClasses/MdsColliveryService.php

```diff
@@ -555,16 +554,16 @@ public function automatedAddCollivery($order_id, $processing = false)
                     $defaults = $this->returnDefaultAddress();
 
                     $address = $this->addColliveryAddress(array(
-                        'company_name' => ($order->shipping_company != '') ? $order->shipping_company : 'Private',
-                        'building' => $order->shipping_address_2,
-                        'street' => $order->shipping_address_1,
-                        'location_type' => $order->shipping_location_type,
-                        'suburb' => $order->shipping_suburb,
-                        'town' => $order->shipping_city,
-                        'full_name' => $order->shipping_first_name . ' ' . $order->shipping_last_name,
-                        'cellphone' => preg_replace('/[^0-9]/', '', $order->shipping_phone),
-                        'email' => str_replace(' ', '', $order->shipping_email),
-                        'custom_id' => $order->user_id,
+                        'company_name' => ($order->get_shipping_company() != '') ? $order->get_shipping_company() : 'Private',
+                        'building' => $order->get_shipping_address_2(),
+                        'street' => $order->get_shipping_address_1(),
+                        'location_type' => $order->get_meta('_shipping_location_type'),
+                        'suburb' => $order->get_meta('_shipping_suburb'),
+                        'town' => $order->get_shipping_city(),
+                        'full_name' => $order->get_shipping_first_name() . ' ' . $order->get_shipping_last_name(),
+                        'cellphone' => preg_replace('/[^0-9]/', '', $order->get_billing_phone()),
```

Comment:
str_replace can become trim($order->get_billing_email()) or removed

Reply:
Why not replace all non digits

Reply:
I just changed the code referencing the data, was not gonna touch what was there.